### PR TITLE
Document adding a configuration for debugging the project

### PR DESCRIPTION
**Pull request by MaciejG604**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -114,13 +114,19 @@ In case of spurious cache-related issues with Gradle build, try one of the follo
...
-## Run & debug
+## Run
 
 To run an instance of IDE with Git Machete IntelliJ Plugin installed from the current source,
 execute `:runIde` Gradle task (`Gradle panel > Tasks > intellij > runIde` or `./gradlew runIde`).
 
 To watch the logs of this IntelliJ instance, run `tail -f build/idea-sandbox/system/log/idea.log`.
 
+## Debug
+
```
Comment (PawelLipski) [inline CONTRIBUTING.md:125]: I'd put a screenshot here, and reduce the size of written instruction

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -114,13 +114,19 @@ In case of spurious cache-related issues with Gradle build, try one of the follo
...
+To debug the plugin using IntelliJ you need to add a `Run Configuration`, to do so go to `Run > Edit Configurations...`.
+Use the plus button in the right upper corner to add a new configuration and choose Gradle from the drop-down list.
+In the `Run` section input `runIde`. <br/>
+Now this new configuration can be chosen in the upper left corner of the IDE, and the Debugging can be started with the Debug button or Ctrl+D (`^D` on Mac).
```
Comment (PawelLipski) [inline CONTRIBUTING.md:129]: Hmmm isn't `Ctrl+D` bound to _Duplicate current line_ by default? 🤔

Comment (MaciejG604) [reply CONTRIBUTING.md:129]: Maybe I was too quick to judge if it's same on Mac and Windows. Will check!

Comment (PawelLipski) [reply CONTRIBUTING.md:129]: I'd ignore Windows, this project is hard to develop on Windows (we tried a couple times... to get `gradle build` to pass was pretty hard). I'm referring to default Linux keymap tho

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -123,9 +123,9 @@ To watch the logs of this IntelliJ instance, run `tail -f build/idea-sandbox/sys
 
 ## Debug
 
-To debug the plugin using IntelliJ you need to add a `Run Configuration`, to do so go to `Run > Edit Configurations...`.
-Use the plus button in the right upper corner to add a new configuration and choose Gradle from the drop-down list.
-In the `Run` section input `runIde`. <br/>
+To debug the plugin using IntelliJ go to `Run > Edit Configurations...` and create a new configuration for Gradle
+![image](https://user-images.githubusercontent.com/61919032/181751590-cc21dd74-f481-4b86-b0d9-b99836669bfd.png)
```
Comment (PawelLipski) [inline CONTRIBUTING.md:127]: Whoops I wouldn't link to an image from outside the repo, it's not clear how long github is gonna keep it around... pls just include the image in the repo and use a relative path, to keep everything self-contained

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**